Added missing includes required by openwrt. Expected to be portable.
--- a/common/Socket_unix.cpp
+++ b/common/Socket_unix.cpp
@@ -8,6 +8,7 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <sys/socket.h>
+#include <sys/select.h> // POSIX.1-2001
 #include <sys/un.h>
 #include <fcntl.h>
 #include <math.h>
--- a/service/ConnectionListener_unix.cpp
+++ b/service/ConnectionListener_unix.cpp
@@ -13,6 +13,7 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <string>
+#include <cstring> // strcpy
 
 #include "service/ConnectionListener.h"
 #include "common/Socket.h"
